pandas/tests/scalar/test_timedelta.py

```diff
@@ -254,6 +254,186 @@ def test_rfloordiv(self):
         with pytest.raises(TypeError):
             ser // td
 
+    def test_td_mod_timedeltalike(self):
```

Comment:
Like `test_rfloordiv` above, I think you can leave out the `_td_` in all test names (the class name already makes clear it is about Timedelta)

Reply:
ok